assets/javascripts/kitten/components/tooltips/tooltip.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import TooltipIcon from './tooltip-icon'
+import ReactTooltip from 'react-tooltip'
+
+export default class Tooltip extends React.Component {
+  render() {
+    return (
+      <div>
+        <TooltipIcon data-tip
```

Comment:
On peut sans doute ajouter `aria-describedby={ this.props.id }` pour relier le bouton au tooltip.